src/client/client.go

```diff
@@ -298,7 +301,42 @@ func getUserMachineAddrAndOpts(cfg *config.Config) (string, []Option, error) {
 	if err != nil {
 		return "", nil, err
 	}
-	return fmt.Sprintf("0.0.0.0:%s", DefaultPachdNodePort), options, nil
+	return "", options, nil
+}
+
+func portForwarder() *PortForwarder {
+	log.Infoln("Attempting to implicitly enable port forwarding...")
```

Comment:
Will this message show on every `pachctl` command? That may get a little too chatty quickly.

Reply:
It shows if `--verbose` is enabled

Reply:
Oh cool, that sounds good.

Reply:
Wait, nevermind. Looks like INFOs are printed without verbose. I'll change the level.

Reply:
Yeah, I'm not quite sure what that logging is hooked up to but I don't think it's hooked into the stdlib package. I think you can get away with not logging this even in verbose. As long as errors get logged if it fails it's fine if it succeeds in the background silently.

Reply:
It uses logrus, but it's configured kinda weirdly because of etcd: https://github.com/pachyderm/pachyderm/blob/1ce30d389b28c6ea6a7e2989ae1cc5e572f362ac/src/server/cmd/pachctl/cmd/cmd.go#L167-L185